### ISSUE TITLE
Question regarding underscore

### ISSUE THREAD

**Issue by nicola** [linkeddata/ldnode#22]
Underscore is an awesome library, but I was wondering if we really need it.

I was having a look at https://github.com/linkeddata/ldnode/blob/master/acl.js and I have seen that we are using `_.each`, which is the equivalent of `obj.forEach` that I believe it is also more readable.

Is there any reason for this particular choice?

**Comment by martinmr** [comment]
I tried loooking for a native way to do that but I didn't find it because I was looking for something along the lines of "for elem in object", so I got the impression that there was not a native way. I agree that underscore is unnecessary if there is a native alternative. I'll change the code accordingly.

**Comment by nicola** [comment]
Perfect!
arr.forEach()

- Typed on a phone by Nicola himself




On Fri, Jun 26, 2015 at 11:31 AM -0700, "Martin Martinez Rivera" <notifications@github.com> wrote:












I tried loooking for a native way to do that but I didn't find it because I was looking for something along the lines of "for elem in object", so I got the impression that there was not a native way. I agree that underscore is unnecessary if there is a native alternative. I'll change the code accordingly.
